openapi.yaml

```diff
@@ -0,0 +1,92 @@
+swagger: '2.0'
+info:
+  description: Endpoint for perfoming OAuth2 operations
+  title: OAuth2
+  version: v1
+  contact:
+    name: IS Data Architecture Team
+    url: https://is.oregonstate.edu/data-architecture
+    email: isdataarchitecture@oregonstate.edu
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+schemes:
+  - https
+host: api.oregonstate.edu
+produces:
+  - application/json
+consumes:
+  - application/x-www-form-urlencoded
+paths:
+  /oauth2/token:
+    post:
+      description: Get an access token to make API calls. Part of OAuth2 client credentials flow.
+      consumes:
+        - application/x-www-form-urlencoded
+      parameters:
+        - name: client_id
+          type: string
+          in: formData
+          description: OAuth Client ID for the application (also called "Consumer Key" in the developer portal)
+        - name: client_secret
+          type: string
+          in: formData
+          description: OAuth Client Secret for the application (also called "Consumer Secret" in the developer portal)
+        - name: grant_type
+          type: string
+          enum: [client_credentials]
```

Comment:
not sure if it's possible to have values besides "client_credientials" for this or if we'd want people to use other values

Reply:
Could also be `refresh_token`.